src/questions/derivatives.yaml

```diff
@@ -0,0 +1,948 @@
+      key: D
+- uuid: ad7116fc-e76e-4e5b-b98f-55cea5c7e9e3
+  type: SingleChoice
+  content: Financial Derivatives consist of-
+  hint: NoHint
+  explanation: Financial Derivatives consist of Forwards, Futures, Options, Swaps
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Stocks
+      key: A
+    - content: Bonds
+      key: B
+    - content: Futures
+      key: C
+    - content: None of the above 
+      key: D
+- uuid: 16f39eef-f4d9-47dc-9c9d-2f3b34dea7f9
+  type: SingleChoice
+  content: Porfolio Hedging is done by bank manager to- 
+  hint: NoHint
+  explanation: Hedging is done primarily for reducing the interest rate risk.
+  answerKeys:
+    - A
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Reduce interest rate risk.
+      key: A
+    - content: Increase reinvestment risk
+      key: B
+    - content: Increase exchange rate risk.
+      key: C
+    - content: Increase the probability of gains.
+      key: D
+- uuid: b59e8a13-3f22-4b14-9f27-c9050ce63a62
+  type: SingleChoice
+  content: Payoffs in financial derivatives are connected to-
+  hint: Financial derivatives are issued on the assets previously they are derived from.
+  explanation: explanation
+  answerKeys:
+    - C
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Securities that will be issued in the future.
+      key: A
+    - content: The volatility of interest rates.
+      key: B
+    - content: Previously issued securities.
+      key: C
+    - content: Government standardising the return rates.
+      key: D
+- uuid: da5f00bd-a592-4463-9508-bdf2c85a9741
+  type: SingleChoice
+  content: To hedge a risk for long position we should-
+  hint: NoHint
+  explanation: Hedging risk for long position can be attained by taking a short position 
+  answerKeys:
+    - B
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Medium"
+  choices:
+    - content: Take a long position. 
+      key: A
+    - content: Take a short position.
+      key: B
+    - content: Take alternate long and short positions
+      key: C
+    - content: Take a neutral position.
+      key: D
+- uuid: 82424bfc-264d-49e7-a0f7-26c9e78b516a
+  type: MultipleChoice
+  content: An asset in a derivative can represent?
+  hint: NoHint
+  explanation: An asset can represent a commodity, a bond, a stock, a cryptocurrency and many more.
+  answerKeys:
+    - A
+    - B
+    - D
+  subTopics:
+    - "introduction-to-derivatives"
+  difficultyLevel: "Low"
+  choices:
+    - content: Commodity
+      key: A
+    - content: Bond
+      key: B
+    - content: IOU Document
+      key: C
+    - content: Stock
+      key: D
+- uuid: 84970ae5-4563-4404-9d8b-b59068386f25
+  type: MultipleChoice
+  content: When a bank manager hedges his portfolio by selling futures, he
```

Comment:
We need to update the question with the exact position information also

Reply:
Done